### PR TITLE
859714- Prepared UG document for more command feature in QAT

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
Hi @SubashiniMahendran 

Prepared UG document for more command feature in QAT. Please review and merge the changes

Regards
Yuvaraj M

### CODE REVIEW

File: winui/Ribbon/Ribbon-QAT.md [hunk 1]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Add items to QAT customize dialog window
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:124]: Adding items using QAT Customize dialog window

File: winui/Ribbon/Ribbon-QAT.md [hunk 2]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+
+To open QAT Customize Window, select `MoreCommands` option from the ContextMenu of the QAT. In the QAT Customized dialog Window, the list of Commands is available. The Commands can be filtered only from the Particular tab by using `Choose commands from` option. Then, select the Command to add to the QuickAccessToolBar and add commands to the right Pane of the Quick Access ToolBar Dialog by clicking Add Button. Finally click OK.
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:127]: To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control. 

By using `Choose commands from` option, you can filter the command based on ribbon tabs. Then, select the desired command and click `Add` button to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.

File: winui/Ribbon/Ribbon-QAT.md [hunk 3]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+![Adding WinUI Ribbon Items to QAT Customize Window](Ribbon-QAT-images/winui-ribbon-items-in-more-commands.png)
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:128]: ![MoreCommands option in QAT drop-down context menu] (Ribbon-QAT-images/winui-ribbon-more-commands-option.png)

Modify image name and content as per the above suggestions.

File: winui/Ribbon/Ribbon-QAT.md [hunk 4]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+
+![Adding WinUI Ribbon Items to QAT Customize Window](Ribbon-QAT-images/winui-ribbon-items-to-customize-window.png)
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:131]: ![Select commands in QAT dialog window] (Ribbon-QAT-images/winui-ribbon-select-command-in-qat-dialog-window.png)

File: winui/Ribbon/Ribbon-QAT.md [hunk 5]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+![Adding WinUI Ribbon Items to QuickAccessToolbar](Ribbon-QAT-images/winui-ribbon-item-to-quick-access-toolbar.png)
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:133]: ![Adding command to right pane of QAT dialog window] (Ribbon-QAT-images/winui-ribbon-add-command-to-right-pane-in-qat-dialog-window.png)

File: winui/Ribbon/Ribbon-QAT.md [hunk 6]
```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+Finally, the Items gets displayed in the QAT
+
+![Added WinUI Ribbon Items in QuickAccessToolbar](Ribbon-QAT-images/winui-ribbon-items-in-quick-access-toolbar.png)
+
```
Comment (SubashiniMahendran) [inline winui/Ribbon/Ribbon-QAT.md:137]: ![Add commands to WinUI Ribbon QuickAccessToolbar] (Ribbon-QAT-images/winui-ribbon-add-commands-to-qat.png)

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: winui/Ribbon/Ribbon-QAT.md (hunk 1 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:124]: Modified

File: winui/Ribbon/Ribbon-QAT.md (hunk 6 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:137]: updated

File: winui/Ribbon/Ribbon-QAT.md (hunk 3 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:128]: updated

File: winui/Ribbon/Ribbon-QAT.md (hunk 4 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:131]: updated

File: winui/Ribbon/Ribbon-QAT.md (hunk 5 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:133]: updated

File: winui/Ribbon/Ribbon-QAT.md (hunk 2 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md:127]: modified

File: winui/Ribbon/Ribbon-QAT.md [hunk 7]
```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Adding items using QAT Customize dialog window
+
+To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.
```
Comment (Chrispineagunja) [inline winui/Ribbon/Ribbon-QAT.md:127]: To open the QAT customize dialog window, select the `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.

File: winui/Ribbon/Ribbon-QAT.md [hunk 8]
```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+
+By using `Choose commands from` option, you can filter the command based on ribbon tabs. Then, select the desired command and click `Add` button to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.
```
Comment (Chrispineagunja) [inline winui/Ribbon/Ribbon-QAT.md:129]: Filter the command based on ribbon tabs by using the `Choose commands from` option. Then, select the desired command and click `Add` to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.

File: winui/Ribbon/Ribbon-QAT.md [hunk 9]
```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
...
+
+![MoreCommands option in QAT drop-down context menu](Ribbon-QAT-images/winui-ribbon-more-commands-option.png)
+
+![Select commands in QAT dialog window](Ribbon-QAT-images/winui-ribbon-select-command-in-qat-dialog-window.png)
+
+![Adding command to right pane of QAT dialog window](Ribbon-QAT-images/winui-ribbon-add-command-to-right-pane-in-qat-dialog-window.png)
+
+Finally, the Items gets displayed in the QAT
```
Comment (Chrispineagunja) [inline winui/Ribbon/Ribbon-QAT.md:137]: Finally, the Items are displayed in the QAT.

File: winui/Ribbon/Ribbon-QAT.md [hunk 10]
```diff
@@ -131,6 +148,12 @@ To remove an items from Quick Access Toolbar (QAT), right-click the required QAT
 
 ![WinUI Ribbon QAT item removed](Ribbon-QAT-images/winui-ribbon-qat-item-removed.png)
 
+QAT item can also be removed by using Customize window, just select the QAT item in the right panel and select `Remove`.
```
Comment (Chrispineagunja) [inline winui/Ribbon/Ribbon-QAT.md:151]: QAT items can also be removed by using the Customize window. Select the QAT item in the right panel, then select `Remove`.

**Review by Chrispineagunja — APPROVED** [review]
Content review completed

File: winui/Ribbon/Ribbon-QAT.md (hunk 10 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md]: Updated

File: winui/Ribbon/Ribbon-QAT.md (hunk 9 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md]: modified

File: winui/Ribbon/Ribbon-QAT.md (hunk 8 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md]: modified

File: winui/Ribbon/Ribbon-QAT.md (hunk 7 above)
Comment (YuvarajMasilamani) [reply winui/Ribbon/Ribbon-QAT.md]: modified

**Review by SubashiniMahendran — APPROVED** [review]

**Pull request merged**